src/sentry/integrations/github/issues.py

```diff
@@ -0,0 +1,166 @@
+        else:
+            repo_choices = [(repo['full_name'], repo['full_name']) for repo in repos]
+
+        params = kwargs.get('params', {})
+        repo = params.get('repo')
+        default_repo = repo or repo_choices[0][0]
+        assignees = self.get_allowed_assignees(default_repo)
+
+        return [
+            {
+                'name': 'repo',
+                'label': 'GitHub Repository',
+                'type': 'select',
+                'default': default_repo,
+                'choices': repo_choices,
+                'updatesForm': True,
+            }
+        ] + fields + [
+            {
+                'name': 'assignee',
+                'label': 'Assignee',
+                'default': '',
+                'type': 'select',
+                'required': False,
+                'choices': assignees,
+            }
+        ]
+
+    def create_issue(self, data, **kwargs):
+        client = self.get_client()
+
+        repo = data.get('repo')
+
+        if not repo:
+            raise IntegrationError('repo kwarg must be provided')
+
+        try:
+            issue = client.create_issue(
+                repo=repo,
+                data={
+                    'title': data['title'],
+                    'body': data['description'],
+                    'assignee': data.get('assignee'),
+                })
+        except ApiError as e:
+            raise IntegrationError(self.message_from_error(e))
+
+        return {
+            'key': issue['number'],
+            'title': issue['title'],
+            'description': issue['body'],
+            'repo': repo,
+        }
+
+    def get_link_issue_config(self, group, **kwargs):
+        try:
+            repos = self.get_repositories()
+        except ApiError:
+            repo_choices = [(' ', ' ')]
+        else:
+            repo_choices = [(repo['full_name'], repo['full_name']) for repo in repos]
+
+        params = kwargs.get('params', {})
+        repo = params.get('repo')
+        default_repo = repo or repo_choices[0][0]
+        issues = self.get_repo_issues(default_repo)
+
+        return [
+            {
+                'name': 'repo',
+                'label': 'GitHub Repository',
+                'type': 'select',
+                'default': default_repo,
+                'choices': repo_choices,
+                'updatesForm': True,
+            },
+            {
+                'name': 'externalIssue',
+                'label': 'Issue',
+                'default': '',
+                'type': 'select',
+                'choices': issues,
+
+            },
+            {
+                'name': 'comment',
+                'label': 'Comment',
+                'default': '',
+                'type': 'textarea',
+                'required': False,
+                'help': ('Leave blank if you don\'t want to '
+                         'add a comment to the GitHub issue.'),
+            }
+        ]
+
+    def get_issue(self, data):
```

Comment:
oh actually one other thing you'll need to update is `get_issue` should have the same signature for all of these + the base class. so you should either update them to take the data object or since this already takes kwargs, you could make this `def get_issue(self, issue_id, **kwargs)` and just do `data=kwargs['data']`